pkg/libvirttools/virtualization.go

```diff
@@ -91,10 +122,39 @@ func (ds *domainSettings) createDomain(config *types.VMConfig) *libvirtxml.Domai
 		emulator = noKvmEmulator
 	}
 
+	var deviceIds []pciInfos
+
+	for _, env := range config.Environment {
+		if strings.Contains(env.Key, "QAT") {
+			devinfo := getPciInfo(env.Value)
+			deviceIds = append(deviceIds, devinfo)
+		}
+	}
+	var hostdevs []libvirtxml.DomainHostdev
+	if len(deviceIds) > 0 {
+		for i := 0; i < len(deviceIds); i++ {
```

Comment:
Instead of using c/c++ construction - please use more natural in golang `for _, device := range(deviceIds) { ...`.

Reply:
Oh, got it. Good suggestions. Thanks a lot.